### PR TITLE
Added quiz to 4.0.1 about service ecosystems

### PR DESCRIPTION
**Pull request by cfculler**
Rather silly quiz, but gets the user to think about service ecosystems in real-world examples.

Slightly updated service offerings lists on 4.0.1

### CODE REVIEW

**Review by jburns24 — CHANGES_REQUESTED** [review]
🅱️roke!

Looks like you linked an empty quiz

image

**Comment by cfculler** [comment]
Weird, not too sure how I managed that. I'll get it taken care of soon

**Comment by cfculler** [comment]
Now that I'm rereading this quiz, maybe I add links to wrong answers about what that type of service ecosystem is. That being said, this quiz is immediately below the list of service ecosystems, so I suppose it's not really necessary. I'll add them in if you think it adds value here 👍

File: src/quizzes/chapter-4/4.0.1/service-ecosystems-quiz.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const rawQuizdown = `
+
+# You are working on your senior project and are designing a basic web app. Currently you are at the early stages of development and aren't hosting your project anywhere (ie you are using localhost to run)<br>What service ecosystem are you currently experiencing?
```
Comment (jburns24) [inline src/quizzes/chapter-4/4.0.1/service-ecosystems-quiz.js:3]:
```suggestion
# You are working on a new personal project and are designing a basic web app. Currently you are at the early stages of development and aren't hosting your project anywhere (ie you are using localhost to run)<br>What service ecosystem are you currently experiencing?
```

Remember this is open source and while geared towards our program not everyone is a senior in college :)

**Review by jburns24 — APPROVED** [review]

**Pull request merged**